lib/EasyPost/Http/Requestor.php

```diff
@@ -1,43 +1,37 @@
 <?php
 
-namespace EasyPost;
+namespace EasyPost\Http;
 
+use EasyPost\Constant\Constant;
+use EasyPost\EasypostObject;
+use EasyPost\Exception\Error;
 use GuzzleHttp\Client;
 
 class Requestor
 {
-    /**
-     * @var string
-     */
-    private $supportEmail = 'support@easypost.com';
-
-    /**
-     * @var string
-     */
-    public $apiKey;
-
     /**
      * Constructor.
      *
-     * @param string $apiKey
+     * @param EasyPostClient $client
      */
-    public function __construct($apiKey = null)
+    public function __construct($client)
     {
-        $this->_apiKey = $apiKey;
+        $this->client = $client;
     }
 
     /**
      * Get the API URL.
      *
      * @param string $url
+     * @param bool $beta
      * @return string
      */
-    public static function apiUrl($url = '', $beta = false)
+    public function absoluteUrl($url = '', $beta = false)
```

Comment:
```suggestion
    private function absoluteUrl($url = '', $beta = false)
```
This function should be `private`, it's only being used in `Requestor` file

Reply:
Nice catch, fixed